WeatherMonitoringSystem/Domain/Bots/RainBot.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+using WeatherMonitoringSystem.Config;
+using WeatherMonitoringSystem.Core;
+using WeatherMonitoringSystem.Domain.Config;
+
+namespace WeatherMonitoringSystem.Bots;
+
+public class RainBot : IWeatherBot
+{
+    private readonly BotConfig _config;
+    private bool _isActivated;
+
+    public RainBot()
+    {
+        _config = ConfigurationManager.Instance.RainBot;
+    }
+
+    public bool Update(WeatherData data, out string message)
```

Comment:
I know we said to not return the message, but still we shouldn't return it as `out` :D 
The idea is that you can have a message saved per bot, and in `UserInteraction` you can loop through the bots and print the state.